src/Middleware/Authorization.ts

```diff
@@ -0,0 +1,23 @@
+import express from "express";
+
+import {ServiceError} from "../Services/ServiceError";
+// import {JwtPayload} from "../Types/JwtPayloadType";
+import config from "../Config/Config";
+import * as jwt from "jsonwebtoken";
+import {JwtPayload} from "../Types/JwtPayloadType";
+
+export async function Authorize(
```

Comment:
Authorization or AuthorizeUser would be even better

Reply:
also, if other middlewares are camelCase, then use it here too

Reply:
AuthorizeUser it is :)